nvidia-video-G06.spec

```diff
@@ -652,9 +658,9 @@ fi
 %dir %{_datadir}/glvnd
 %dir %{_datadir}/glvnd/egl_vendor.d
 %{_datadir}/glvnd/egl_vendor.d/10_nvidia.json
+%if 0%{?suse_version} < 1550
```

Comment:
I think this if is missing a closing?

Reply:
> I think this if is missing a closing?

No, I moved this `if` two lines above since build failed because it tries to package non-existent directories.
```
%dir %{_datadir}/egl
%dir %{_datadir}/egl/egl_external_platform.d
```